### DISCUSSION TITLE
Using .pkg installer vs script

### DISCUSSION THREAD

**Discussion by quackerex** [Homebrew/discussions#4713] [Getting started]
### Output of `brew config`

```shell
NA
```

### Output of `brew doctor`

```shell
NA
```

### Description of issue

What is the difference between installing brew via the .pkg installer as opposed to using the script? Are there discernible benefits or drawbacks associated with either method?

**Comment by SMillerDev** [comment]
The pkg will only work on macOS. That's the only drawback I can think of.

**Comment by carlocab** [comment]
For a personal machine which you have administrator access to, they should be equivalent.

If you're administrating a number of Macs (e.g. for your workplace), then deploying the pkg installer may work out to be easier.

**Comment by mima-mima** [comment]
deploying brew via a package manager to multiple macs has always been a significant challenge filled with workarounds that cange release to release. PKG deployment should make admins' lives massively easier.
